src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -62,7 +62,7 @@
     public static final String CEXT_MIME_TYPE = "application/x-ruby-cext-library";
     public static final String CEXT_EXTENSION = ".su";
 
-    public final boolean SINGLE_THREADED = Boolean.getBoolean("truffleruby.single_threaded");
+    public final boolean SINGLE_THREADED = Boolean.parseBoolean("truffleruby.single_threaded");
```

Comment:
This line's wrong! `Boolean.parseBoolean` instead of `Boolean.getBoolean`.

Reply:
Indeed.